public/app/features/logs/components/InfiniteScroll.tsx

```diff
@@ -57,8 +66,20 @@ export const InfiniteScroll = ({
         return;
       }
       event.stopImmediatePropagation();
+
+      // Give the user a chance to reach the top or bottom before triggering infinite scrolling requests
+      const onScrollingEdge = isOnEdge(lastScroll.current, scrollElement, onEdge, debouncedOnEdge, () => {
```

Comment:
Isn't the `onEdgeCallback` callback missing to call `setOnEdge(true)`?

Reply:
The debounced flavor always set it to true. There's no need to debounce setting to false, at least in this version.

Reply:
Oh right, I missed that!